docs/sources/k6/v0.57.x/shared/experimental-module.md

```diff
@@ -7,7 +7,7 @@ title: Experimental module admonition
 This is an experimental module.
 <br>
 <br>
-While we intend to keep experimental modules as stable as possible, we may need to introduce breaking changes. This could happen at future k6 releases until the module becomes fully stable and graduates as a k6 core module. For more information, refer to the [extension graduation process]({{< relref "../extensions/explanations/extension-graduation" >}}).
+While we intend to keep experimental modules as stable as possible, we may need to introduce breaking changes. This could happen at future k6 releases until the module becomes fully stable and graduates as a k6 core module. For more information, refer to the [extension graduation process](../../extensions/explanations/extension-graduation/).
```

Comment:
```suggestion
While we intend to keep experimental modules as stable as possible, we may need to introduce breaking changes. This could happen at future k6 releases until the module becomes fully stable and graduates as a k6 core module. For more information, refer to the [extension graduation process](https://grafana.com/docs/k6/<K6_VERSION>/extensions/explanations/extension-graduation/).
```